sound/soc/sof/intel/hda-stream.c

```diff
@@ -486,7 +486,7 @@ irqreturn_t hda_dsp_stream_threaded_handler(int irq, void *context)
 
 			/* Inform ALSA only in case not do that with IPC */
 			if (sof_hda->no_ipc_position)
-				snd_pcm_period_elapsed(s->substream);
+				snd_sof_pcm_elapsed(s->substream);
```

Comment:
I really don't like all this nonsense of no_ipc_position. We are circling on the same problems on how to report the position...

Reply:
I can imagine, :) but that has no relationship with this PR, right?